Makefile

```diff
@@ -0,0 +1,51 @@
+init:
```

Comment:
I would add following lines on top:

```
%:
        @:

args = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
```

that is useful if an user calls `make` without arguments (it shows all available arguments)

and then
```
.PHONY: init run start stop install install_prod [...] # please complete this list...
```

See [SO answer](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile)

Reply:
the first three lines doesn't work :S

Reply:
are you using a tab before `@`? I think that you can't use spaces to indent a Makefile

Reply:
yep! 
![Schermata 2020-03-24 alle 15 21 48](https://user-images.githubusercontent.com/190820/77436001-3efd1a80-6de3-11ea-8d9d-4973a59f0629.png)